torch/_dynamo/config.py

```diff
@@ -50,13 +49,13 @@
 
 # Assume these functions return constants
 constant_functions = {
-    torch.jit.is_scripting: False,
-    torch.jit.is_tracing: False,
-    torch._C._get_tracing_state: None,
-    torch.fx._symbolic_trace.is_fx_tracing: False,
-    torch.onnx.is_in_onnx_export: False,
-    external_utils.is_compiling: True,
-    torch._utils.is_compiling: True,
+    "torch.jit.is_scripting": False,
```

Comment:
This was to make the config hash more determinstic. Please advise if this is ok @voznesenskym @ezyang

Reply:
I think this change is too risky after all. Let's put it as a TODO in a standalone PR.